stl/inc/atomic

```diff
@@ -161,7 +161,7 @@ _Ty kill_dependency(_Ty _Arg) noexcept { // "magic" template that kills dependen
 // FUNCTION _Check_memory_order
 inline void _Check_memory_order(const memory_order _Order) noexcept {
     // check that _Order is a valid memory_order
-    if (static_cast<unsigned int>(_Order) > memory_order_seq_cst) {
+    if (static_cast<int>(_Order) > static_cast<int>(memory_order_seq_cst)) {

```

Comment:
These should cast to `unsigned int` to ensure that `_Order` is actually in `[0, memory_order_seq_cst]`; `_Order` could otherwise be `memory_order{-999}` which is certainly `<= static_cast<int>(memory_order_seq_cst)`, but just as certainly not a valid memory order.

Reply:
I pushed this change to the branch.